runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1570,6 +1539,16 @@ TR_ResolvedJ9JITServerMethod::createResolvedMethodFromJ9MethodMirror(TR_Resolved
    packMethodInfo(methodInfo, resolvedMethod, fe);
    }
 
+bool
+TR_ResolvedJ9JITServerMethod::verifyCache(TR_ResolvedJ9JITServerMethodInfo clientMethodInfo, TR_ResolvedJ9JITServerMethodInfo cachedMethodInfo)
+   {
+   if (std::get<0>(clientMethodInfo).isInterpreted)
```

Comment:
If the method from the client is interpreted, why is the check succeeding trivially?
I think I comment is warranted here.

Reply:
It's succeeding because if a method is still interpreted, then some of the parameters might differ from the cached ones, but since we are only caching for the duration of one compilation, we will ignore the discrepancy.